### ISSUE TITLE
Encoding.GetBytes may return wrong value

### ISSUE THREAD

**Issue by takuya-takeuchi** [takuya-takeuchi/DlibDotNet#160]
# Summary of your issue

When call ShapePredictor.Deserialize, native library throw exception.
It looks like to be wrong value passed to native library.

# Environment

Japanese Windows

# What did you do when you faced the problem?

Failed to call ShapePredictor.Deserialize

## Example code:

```
ShapePredictor.Deserialize("C:\Users\XXXXX\Downloads\shape_predictor_68_face_landmarks.dat")
```

## Output:

N/A

## What did you intend to be?

Success to call ShapePredictor.Deserialize

## Reference

https://qiita.com/kumaS-kumachan/items/326ec6afa201c85d0a34#comment-bfd54d47be9d5014e754

**Comment by takuya-takeuchi** [comment]
````
using System;
using System.Collections.Generic;
using System.Linq;
using DlibDotNet;

namespace Test
{

    internal class Program
    {

        private static void Main(string[] args)
        {
            var sp = ShapePredictor.Deserialize(@"C:\Users\TAKUYA\Downloads\shape_predictor_5_face_landmarks.dat");
            Console.WriteLine(sp);
        }
        
	}
	
}
````

![image](https://user-images.githubusercontent.com/6241854/69913028-f73e3600-1475-11ea-8d19-d7de52dd17af.png)

It looks good to me....

**Comment by takuya-takeuchi** [comment]
However, I do not pass the length of file_name.
So it should be

* pass the length of path string
* create std::string object from `const char* path` and `int length`

**Comment by takuya-takeuchi** [comment]
We must update all usage of GetBytes. 73 points. :(

**Comment by takuya-takeuchi** [comment]
fix on 488301b9d7b04514def12ddf392fa3af1bbe4cdc
But I never face to this issue. I need someone confirm it.

**Comment by takuya-takeuchi** [comment]
close. Issue original reporter confirmed fix.
https://qiita.com/kumaS-kumachan/items/326ec6afa201c85d0a34
